client/src/workspace/layouts/AppsViewLayout.tsx

```diff
@@ -1,53 +1,90 @@
 import React, { Suspense } from 'react';
 import { ErrorBoundary } from 'react-error-boundary';
 import { Outlet } from 'react-router-dom';
-import { Alert, Layout } from 'antd';
+import { Alert, Layout, Flex } from 'antd';
 import { Spinner } from '@client/common-components';
-import { AppsSubmissionForm, useGetAppParams } from '@client/workspace';
-import { useAppsListing } from '@client/hooks';
+import { AppsSubmissionForm, useGetAppParams, findAppById, AppIcon } from '@client/workspace';
+import { useAppsListing, useGetAppsSuspense } from '@client/hooks';
 import parse from 'html-react-parser';
 import styles from './layout.module.css';
 
 export const AppsViewLayout: React.FC = () => {
   const { appId, appVersion } = useGetAppParams();
+  const { data: app } = useGetAppsSuspense({ appId, appVersion });
   const { data } = useAppsListing();
+  
+  const icon =
+    findAppById(data, app.definition.id)?.icon ??
+    (app.definition.notes.icon || 'Generic-App');
+  const userGuideLink =
+    findAppById(data, app.definition.id)?.userGuideLink ??
+    (app.definition.notes.helpUrl);
+
   const htmlApp = data?.htmlDefinitions[appId];
   const key = `${appId}-${appVersion}`;
+
+  const { Header } = Layout;
+  const headerStyle = {
+    background: 'transparent',
+    paddingLeft: 0,
+    paddingRight: 0,
+    borderBottom: '1px solid #707070',
+    fontSize: 16,
+  };
+
+
   return (
     <>
-      {htmlApp ? (
-        <div className={styles['overflow']}>
-          {parse(htmlApp.html as string)}
-        </div>
-      ) : (
-        <ErrorBoundary
-          key={key}
-          fallbackRender={({ error }) =>
-            error && (
-              <div id="appDetail-wrapper">
-                <Alert
-                  message={error?.response?.data?.message ?? error.message}
-                  type="error"
-                  showIcon
-                  style={{ marginTop: 10 }}
-                />
-              </div>
-            )
-          }
-        >
-          <Suspense
-            fallback={
-              <Layout>
-                <Spinner />
-              </Layout>
-            }
+      <Suspense
```

Comment:
Move this suspense to https://github.com/DesignSafe-CI/portal/blob/41cd28a2f14a9339eef8c966f047dff536f761c2/client/src/workspace/workspaceRouter.tsx#L21
and  add <ErrorResponse> wrapper over it.

I tried locally and works.